src/components/ToDoContainer/NewTodoForm.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+const NewTodoForm = (props) => {
+    return (
```

Comment:
If you have a function that has only a return value and nothing before that (like this), you can shorten the definition to something like:
```
const NewTodoForm = ({formSubmitted, ... }) => <form>...</form>
```